atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraClientPoolMetrics.java

```diff
@@ -39,12 +42,16 @@ public class CassandraClientPoolMetrics {
     // Not bundled in with request metrics, as we seek to not produce host-level metrics for economic reasons.
     private final Counter poolExhaustionCounter;
 
+    private final Counter emptyPoolCounter;
+
     public CassandraClientPoolMetrics(MetricsManager metricsManager) {
         this.metricsManager = metricsManager;
         this.aggregateRequestMetrics = new RequestMetrics(metricsManager);
         this.poolExhaustionCounter =
                 metricsManager.registerOrGetCounter(CassandraClientPoolMetrics.class, "pool-exhaustion");
         this.outlierControllers = createOutlierControllers(metricsManager);
+        this.emptyPoolCounter =
+                metricsManager.registerOrGetCounter(CassandraClientPoolMetrics.class, EMPTY_POOL_METRIC_NAME);
```

Comment:
pool-exhaustion is on `CassandraClientPoolMetrics`, so biasing towards that, although I'd prefer to just put this on `CassandraClientPool`